static/fetch_lib.sh

```diff
@@ -3,30 +3,23 @@
 true
 # see https://github.com/koalaman/shellcheck/wiki/Directive
 
-# Functions
-msg_box() {
-    [ -n "$2" ] && local SUBTITLE=" - $2"
-    whiptail --title "$TITLE$SUBTITLE" --msgbox "$1" "$WT_HEIGHT" "$WT_WIDTH"
-}
-
-
```

Comment:
4. I think a msg_box/whiptail is a bit overkill for fetch_lib. And does it work? (TITLE, SUBTITLE, WT_HEIGHT, WT_WIDTH should be empty...